spring-data-cassandra/src/main/java/org/springframework/data/cassandra/core/CassandraOperations.java

```diff
@@ -280,7 +281,40 @@
 	 * @return The entities given.
 	 */
 	<T> List<T> update(List<T> entities, WriteOptions options);
-
+	/**
+	 * Execute the Update Query and convert to the list of entities
+	 * 
+	 * @param Update must not be {@literal null}.
+	 * @param type must not be {@literal null}, mapped entity type.
+	 * @return
+	 */
+	<T> List<T> update(Update update, Class<T> type);
+	/**
+	 * Execute the Update Query and convert to the list of entities
+	 * 
+	 * @param Update must not be {@literal null}.
+	 * @param type must not be {@literal null}, mapped entity type.
+	 * @param options The {@link WriteOptions} to use.
+	 * @return
+	 */
+	<T> List<T> update(Update update, Class<T> type, WriteOptions options);
```

Comment:
`WriteOptions` (ttl, consistency level, retry policy) are already part of the `Update` object so I'd propose to skip this method.